pkg/init/cmd/service/prepare.go

```diff
@@ -195,6 +196,28 @@ func prepareFilesystem(path string, runtime Runtime) error {
 	return nil
 }
 
+// prepareSpec appends spec based on runtime config;
+// NOTE these changes are runtime-only, we don't update config.json after this
```

Comment:
To be honest I was a little surprised by this, maybe it's something that we should fix. It seems like a plausible thing for one to look at the file when debugging it.

Reply:
The filesystem is read only usually, so it cannot just be updated.

Reply:
Ah, obviously... :)